Linked Lists/RevaLinkListGivenAgROUP.py

```diff
@@ -0,0 +1,67 @@
+class Node:
+    def __init__(self,data):
+        self.data=data
+        self.next=None
+class Linkedlist:
+    def __init__(self):
+        self.head=None
+    def insert(self,newNode):
+        if self.head is None:
+            self.head=newNode
+        else:
+            lastNode=self.head
+            while True:
+                if lastNode.next is None:
+                    break
+                else:
+                    lastNode=lastNode.next
+            lastNode.next=newNode
+    def printlist(self):
+        currentNode=self.head
+        while True:

```

Comment:
while( currentNode is not None):
       print(currentNode.data)
       currentNode = currentnode.next